README.md

```
@@ -21,12 +20,13 @@ OpenAIのAgents SDKを使用して、MCPサーバーを利用したAIエージ
 - [⚙️ 環境に応じた設定](#環境に応じた設定)
 - [▶️ 実行方法](#実行方法)
 - [🔑 環境変数](#環境変数)
+- [💫 互換モード](#互換モード)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)